spot_driver_cpp/src/spot_image_sources.cpp

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
+
+#include <spot_driver_cpp/spot_image_sources.hpp>
+#include <spot_driver_cpp/types.hpp>
+#include <tl_expected/expected.hpp>
+
+namespace
+{
+const std::vector<std::string> kImageSourceNames = {
+    "back",
+    "frontleft",
+    "frontright",
+    "left",
+    "right",
+};
+
+constexpr auto kImageSourceHand = "hand";
```

Comment:
Are these names not defined in the Spot C++ api anywhere?

Reply:
The fully-qualified names of the camera sources are defined [here in the SDK library](https://github.com/bdaiinstitute/spot-cpp-sdk/blob/master/cpp/bosdyn/client/image/image_source_names.h), but these prefixes aren't included there. It's useful to have them defined to let us parse the SDK's source names into our own types, especially since the SDK doesn't include functions to convert them the other way.